<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Item</class>
 <widget class="QWidget" name="Item">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>244</width>
    <height>285</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>241</width>
     <height>278</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius: 17px;
background: rgb(255, 255, 255);
</string>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QLabel" name="Price_L">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>240</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(6, 106, 110);
font: 700 14pt &quot;Segoe UI&quot;;
background: none;
padding-bottom: 3px;</string>
    </property>
    <property name="text">
     <string>Price</string>
    </property>
   </widget>
   <widget class="QLabel" name="Image_L">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>221</width>
      <height>131</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">	background-color: rgb(223, 229, 229);
	color: rgb(31, 48, 58);
	border-radius: 5px;
	border-width: 0.5px;
	border-style: solid;
	border-color: rgb(188, 193, 193);
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QTextEdit" name="Description_TE">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>180</y>
      <width>211</width>
      <height>59</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QTextEdit {
    color: rgb(31, 48, 58);
    background: rgb(255, 255, 255);
}

QScrollBar:vertical {
    background: rgb(255, 255, 255); 
    width: 3px;
    border-radius: 2px;
}

QScrollBar::handle:vertical {
    background:  rgb(255, 137, 64);
    min-height: 3px;
    border-radius: 2px;
}

QScrollBar::add-line:vertical {
    background: rgb(255, 255, 255);
}

QScrollBar::sub-line:vertical {
    background: rgb(255, 255, 255);
}

QScrollBar::add-page:vertical, QScrollBar::sub-page:vertical {
    background: rgb(255, 255, 255);
}

QMenu {
    border: 1px solid white;
    border-radius: 5px;
    background-color: rgb(6, 106, 110);
    padding: 7px;
    color: white;
}

QMenu::item {
    padding: 4px;
}

QMenu::item:hover {
    border: none;
    outline: none;
}

QMenu::item:selected {
    background-color: rgb(255, 137, 64);
    border-radius: 5px;
    color: rgb(31, 48, 58);
}

QMenu::separator {
    height: 1px;
    background-color: white;
}</string>
    </property>
    <property name="html">
     <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;meta charset=&quot;utf-8&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
hr { height: 1px; border-width: 0; }
li.unchecked::marker { content: &quot;\2610&quot;; }
li.checked::marker { content: &quot;\2612&quot;; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Segoe UI'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QTreeView" name="treeView">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>240</width>
      <height>277</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="Name_PB">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>150</y>
      <width>221</width>
      <height>24</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(31, 48, 58);
font: 600 13pt &quot;Segoe UI&quot;;
background: none;
padding-bottom: 3px;
text-align: left;</string>
    </property>
    <property name="text">
     <string>Name</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Removeamount_PB">
    <property name="geometry">
     <rect>
      <x>161</x>
      <y>250</y>
      <width>25</width>
      <height>25</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: rgb(190, 190, 190);
    border: none;
    border-radius: 12px;
	padding-bottom: 5px;
	padding-left: 0.5px;
	padding-top: 5px;
	padding-right: 1px;
	image: url(:/icons/images/icons/remove.png);
	font: 600 15pt &quot;Segoe UI&quot;;
}
QPushButton:hover {
    background-color: rgb(7, 132, 136);
}
QPushButton::pressed {
    background-color: rgb(5, 97, 100);
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="Addamount_PB">
    <property name="geometry">
     <rect>
      <x>209</x>
      <y>250</y>
      <width>25</width>
      <height>25</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: rgb(6, 106, 110);
    border: none;
    border-radius: 12px;
	padding-bottom: 5px;
	padding-left: 0.5px;
	padding-top: 5px;
	padding-right: 1px;
	image: url(:/icons/images/icons/plus.png);
	font: 600 15pt &quot;Segoe UI&quot;;
}
QPushButton:hover {
    background-color: rgb(7, 132, 136);
}
QPushButton::pressed {
    background-color: rgb(5, 97, 100);
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="Amount_L">
    <property name="geometry">
     <rect>
      <x>183</x>
      <y>245</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit {
	background: white;
	color: rgb(31, 48, 58);
	font: 600 15pt &quot;Segoe UI&quot;;
}</string>
    </property>
    <property name="text">
     <string>0</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignmentFlag::AlignCenter</set>
    </property>
   </widget>
   <zorder>treeView</zorder>
   <zorder>Price_L</zorder>
   <zorder>Image_L</zorder>
   <zorder>Description_TE</zorder>
   <zorder>Name_PB</zorder>
   <zorder>Amount_L</zorder>
   <zorder>Addamount_PB</zorder>
   <zorder>Removeamount_PB</zorder>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
